### PR TITLE
Update swedish translation

### PR DESCRIPTION
**Pull request by Mr-Bajs**
Mostly just some polishing of existing translations, added capitalisations.

### CODE REVIEW

File: files/lang/sv.po [hunk 1]
```diff
@@ -2882,9 +2885,8 @@ msgstr "Tangentbordsgenvägar"
 msgid "Select Game Language:"
 msgstr "Välj spelspråk:"
 
-#, fuzzy
 msgid "Click to choose the selected language."
-msgstr "Klicka för att gå tillbaka till huvudmenyn."
+msgstr "Klicka för att ange valt stråk."
```
[the remark this answers is not in the document]
Comment (Mr-Bajs) [reply files/lang/sv.po]: you got eyes of a hawk ;)

**Comment by Mr-Bajs** [comment]
> Hi, @Mr-Bajs. Could you please resolve the merge conflicts for this PR?

Sure, but how do I do that?

**Comment by Mr-Bajs** [comment]
> > > Hi, @Mr-Bajs. Could you please resolve the merge conflicts for this PR?
> > 
> > 
> > @zenseii Sure, but how do I do that?
> 
> Scroll down to the bottom of the PR page and click "Resolve conflicts" on the bottom of the build checks.
> 
> Looking at the conflict there is only a conflict between the POT date of your PR and the fheroes2 master. Just choose the most recent one and delete the other one and delete the lines with <<<<<<<, >>>>>> and ===== etc. Then click mark as solved and then merge.

Okey :)

**Comment by ihhub** [comment]
@Mr-Bajs , thank you so much for this update of the translation!

**Pull request merged**